usecases/objects/vector.go

```diff
@@ -0,0 +1,51 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package objects
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-openapi/strfmt"
+	"github.com/semi-technologies/weaviate/entities/additional"
+	"github.com/semi-technologies/weaviate/entities/search"
+)
+
+func (m *Manager) updateRefVector(ctx context.Context,
+	className string, id strfmt.UUID,
+) error {
+	if m.modulesProvider.UsingRef2Vec(className) {
+		parent, err := m.vectorRepo.Object(ctx, className,
+			id, search.SelectProperties{}, additional.Properties{})
+		if err != nil {
+			return fmt.Errorf("find parent '%s/%s': %w",
+				className, id, err)
+		}
+
+		obj := parent.Object()
+
+		if err := m.modulesProvider.UpdateVector(
+			ctx, obj, m.vectorRepo.Object, m.logger); err != nil {
+			return fmt.Errorf("calculate ref vector for '%s/%s': %w",
+				className, id, err)
+		}
+
+		if err := m.vectorRepo.PutObject(ctx, obj, obj.Vector); err != nil {
+			return fmt.Errorf("put object: %s", err)
```

Comment:
here should be `%w` operator instaead of `%s`

Reply:
fixed